app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_origin = command.split()[1]
+    cp_file = command.split()[2]
```

Comment:
Use unpacking and do not perform split operation twice. Before unpacking check that the number of elements after split equals to three